### PR TITLE
NickServ auth implemented.

### LINKED ISSUE #16: add auth against NickServ
The API used for IRC connection does support NickServ, we only need to add some lines to support it and be able to get the data/config from the Configuration file.

### PR DESCRIPTION
**Pull request by gcanosa**
close #16

### CODE REVIEW

**Comment by gcanosa** [comment]
@gresco test it =) and let's merge if it does the job.

**Comment by gpoppino** [comment]
Please, check **SASL** authentication: 
- https://github.com/TheLQ/pircbotx/wiki/Documentation#sasl-authentication

Then, if it fails try to use bot.send().identify().

**Review by gpoppino — CHANGES_REQUESTED** [review]
Please, review the comments I've made to your commit to this branch. SASL support is a must before merging.

File: README.md [hunk 1]
```diff
@@ -4,8 +4,8 @@
 This project has been entirely written in the Java language, so it basically can
 run on computers that are able to run Java code.
 
-We focus on handling memory in the best possible way, keep a clean code,
-stability and reliability.
+We focus on handling memory in the best possible way, keeping a clean code,
+always trying to use best programming practises, stability and reliability.
```
Comment (gpoppino) [inline README.md]: Replace _practises_ for _practices_.

**Comment by gpoppino** [comment]
Feature already merged into master. All commits have been squashed.

**Comment by gcanosa** [comment]
@gresco made a squash first, and then a merge to the master branch. It's been implemented.

**Pull request closed without merging**